webcompat/views.py

```diff
@@ -274,7 +275,16 @@ def contributors():
 @app.route('/tools/cssfixme')
 def cssfixme():
     '''Route for CSS Fix me tool'''
-    return render_template('cssfixme.html')
+
+    url = request.args.get('url')
```

Comment:
Missing test suite, for testing the URLs.
For example, this is crashing the site
http://localhost:5000/tools/cssfixme?url=blah
because blah is not an url. See below.